src/com/lftechnology/java/training/alina/jdbc/dbutils/DbFacade.java

```diff
@@ -0,0 +1,117 @@
+package com.lftechnology.java.training.alina.jdbc.dbutils;
+
+import java.sql.Date;
+import java.sql.DriverManager;
+import java.sql.ParameterMetaData;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+
+import com.lftechnology.java.training.alina.jdbc.domain.Database;
+
+/**
+ * DbFacade consists of database connection related functionalities
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class DbFacade {
+
+    private static final Logger LOGGER = Logger.getLogger(DbFacade.class.getName());
+    private static final String JDBC_DRIVER = "com.mysql.jdbc.Driver";
+    private static final String DB_URL = "jdbc:mysql://127.0.0.1/employee_mgmt?rewriteBatchedStatements=true";
+    private static final String USER = "root";
+    private static final String PASS = "";
+    private static Connection connection;
+
+    /**
+     * Connects to database
+     * 
+     * @return connection {@link Connection}
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static Connection getDbConnection() {
+        if (connection == null) {
+            try {
+                Class.forName(JDBC_DRIVER);
+                connection = DriverManager.getConnection(DB_URL, USER, PASS);
+            } catch (Exception e) {
+                LOGGER.log(Level.WARNING, "Exception Message : {0}", new Object[] { e });
+            }
+        }
+        return connection;
+    }
+
+    /*
+     * Closes database connections
+     */
+    public static void closeDbConnection() throws SQLException {
+        if (connection != null) {
+            connection.close();
+        }
+    }
+
+    /**
+     * Set value on the basis of different instances
+     * 
+     * @param ps
+     * @param args
+     * @return
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static PreparedStatement mapParams(PreparedStatement ps, Object... args) throws SQLException {
+        int i = 1;
+        for (Object arg : args) {
+            if (arg instanceof Date) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00108)